pkgs/development/libraries/poppler/default.nix

```diff
@@ -32,8 +33,20 @@
 
 let
   mkFlag = optset: flag: "-DENABLE_${flag}=${if optset then "on" else "off"}";
+
+  # unclear relationship between test data repo versions and poppler
```

Comment:
Might be convenient to have it in `passthru` for easier access (will also be needed for an update script anyway).